CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Use the following format for additions: ` - VERSION: [feature/patch (if applicab
 - 1.1.21:
```

Comment:
Sorry I just noticed the version in package.json and changelog.md was out of sync. Fixed on master now. This one should be 1.1.22 instead.